internal/compose/compose.go

```diff
@@ -275,49 +275,57 @@ func (p *Project) WaitForHealthy(opts CommandOptions) error {
 		return err
 	}
 
+	timeout := time.After(10 * time.Minute)
+	ticker := time.NewTicker(1 * time.Second)
+
 	containerIDs := strings.Split(strings.TrimSpace(b.String()), "\n")
 	for {
 		if signal.SIGINT() {
 			return errors.New("SIGINT: cancel waiting for policy assigned")
 		}
 
-		healthy := true
-
-		logger.Debugf("Wait for healthy containers: %s", strings.Join(containerIDs, ","))
-		descriptions, err := docker.InspectContainers(containerIDs...)
-		if err != nil {
-			return err
-		}
-
-		for _, containerDescription := range descriptions {
-			logger.Debugf("Container status: %s", containerDescription.String())
+		select {
+		case <-timeout:
+			return errors.New("time out waiting for healthy container")
+		case <-ticker.C:
+			healthy := true
 
-			// No healthcheck defined for service
-			if containerDescription.State.Status == "running" && containerDescription.State.Health == nil {
-				continue
+			logger.Debugf("Wait for healthy containers: %s", strings.Join(containerIDs, ","))
+			descriptions, err := docker.InspectContainers(containerIDs...)
+			if err != nil {
+				return err
 			}
-
-			// Service is up and running and it's healthy
-			if containerDescription.State.Status == "running" && containerDescription.State.Health.Status == "healthy" {
-				continue
+			for _, containerDescription := range descriptions {
```

Comment:
Let's extract this change around ticker to a separate PR and focus here on improving the terraform Docker image.

Reply:
Done https://github.com/elastic/elastic-package/pull/687

Reply:
So, could you please remove it from this PR?